### DISCUSSION TITLE
Multi domain store setup

### DISCUSSION THREAD

**Discussion by musabatas** [magenx/Magento-2-server-installation#75] [Q&A] [answered]
I would like to configure Magento can run a multi-domain store setup. Which is the best way? Should I make changes in magento2.conf or create separate conf files for each domain?
I have tried configuring magento2.conf with multiple domains but the additional store domains redirect to the main store domain for the first visit.

**Comment by gfxguru** [comment]
I'm trying to do the same. Following

**Comment by magenx** [comment, marked as answer]
>you have to create magento store/website/view

https://devdocs.magento.com/guides/v2.4/config-guide/multi-site/ms_over.html

>use code in nginx config 

https://github.com/magenx/Magento-nginx-config/blob/master/magento2/conf_m2/maps.conf#L48
https://github.com/magenx/Magento-nginx-config/blob/master/magento2/sites-available/magento2.conf#L99
